// Copyright 2012 The Chromium Authors
// Use of this source code is governed by a BSD-style license that can be
// found in the LICENSE file.
//
// Protocol messages used only for testing.

syntax = "proto2";

option java_multiple_files = true;
option java_package = "org.chromium.components.sync.protocol";

option optimize_for = LITE_RUNTIME;

package sync_pb;

message UnknownFieldsTestA {
  required bool foo = 1;
}

message UnknownFieldsTestB {
  required bool foo = 1;
  required bool bar = 2;
}
